cli/internal/model/serverlist.go

```diff
@@ -17,8 +18,40 @@ type ServerList struct {
 	Servers  map[ext.ServerUUID]ext.Server
 }
 
-func SaveToFile(serverList ServerList, path string) error {
-	data, err := json.Marshal(serverList)
+type Cache struct {
+	ServerList
+	// last vault access to entity
+	TenantsTimestamps  map[iam.TenantUUID]time.Time
+	ProjectsTimestamps map[iam.ProjectUUID]time.Time
+	ServersTimestamps  map[ext.ServerUUID]time.Time
+	// ttl of entity
+	TTL time.Duration
```

Comment:
Looks like cache-specific fields (time maps and TTL) are not used externally. Should they be exported? I suggest making them unexported.

Reply:
done